modules/ROOT/pages/apicat-create-descriptor-file-manually.adoc

```diff
@@ -0,0 +1,226 @@
+
+Following is an example `triggerConditions` section:
+
+---
+ triggerConditions: 
+   branches:
+     - main
+     - release/(.*)
+   tags:
+     - support
+     - release/(.*)
+
+---
+
+The preceding example enables a run of the `api-catalog publish-asset` command to publish the APIs described by the descriptor file only if the command has all of the following `--trigger-criteria` option values: 
+
+* `branches` are `main` or have `release/` in the text
+* `tags` are `support` or have `release/` in the text
+
+If the `triggerConditions` section is included in the descriptor file, no APIS are published by the `api-catalog publish-asset` command unless all of the trigger conditions are matched by `--trigger-criteria` option values. For an example publish command using the `--trigger-criteria` option, see 
+xref:apicat-publish-using-api-catalog-cli.adoc#example-trigger[Trigger the Publish Using Conditions].
+
+[[version-strategy-conditions]]
+== versionStrategyConditions
+
+The `versionStrategyConditions` section of the YAML file enables you to optionally set conditions to determine the version strategy of the set of APIs described in the descriptor file. If you add this section to the descriptor file, the version strategy of the APIs are set when the `api-catalog publish-asset` command is run with `--version-strategy-criteria` option. This enables you use the same `api-catalog publish-asset` command in your CI/CD scripts for each of your branches or environments and pass parameters to the command to control the version strategy of the APIs that are published. 
+
+You can set version strategy conditions for specific APIs or for all APIs described in the descriptor file (global). If both types of conditions are used, the specific override the global. 
+
+Following is an example `versionStrategyConditions` section:
+
+----
+    versionStrategyConditions:
+      minorIncrease:
+        branches:
+          - main
+        tags:
+          - support
+      snapshot:
+        branches:
+          - develop
+----
+
+== contact
+
+The `contact` section of the schema includes the name and email ID for main contact for the autocataloging. If none are included, these default to `unknown`.   
+
+== projects 
+
+The `projects` section of the schema lists the API projects to be cataloged along with the details needed to catalog them. 
+
+spec:: The location of the specification file that describes the API. 
+
+assetId:: Specifies the ID of the asset to be published in Exchange. 
+
+documentation:: A list of key and value pairs that specify the name of the documentation page and file location for the API.
+These pairs define the order of the documentation pages in Anypoint Platform.
++
+Format: <Name of the documentation page>:<File Location>
++
+If this option is not specified, no documentation pages are imported. 
```

Comment:
Can each of these with similar construction be rewritten to something like:

If you don't specify this option, Exchange (or whatever) does not import documentation pages